docs/n00b-advanced.md

```diff
@@ -0,0 +1,21 @@
+# Advanced n00b mermaid (Coming soon..)
```

Comment:
Maybe just show it as strikethrough in the sidebar link or some other way? Clicking into a page to find out it actually is not ready is a bit annoying. If it's not done, it's better if we don't have a partial/confusing page.

Reply:
Thanks! I agree with you completely, both assessment and thoughts on rewriting. Writing from a first person perspective is not optimal in any sense, but seemed like the quickest way to jot down lessons learned. Integrating with the full docs and in a depersonalized way would be a desirable end state.